app/assets/javascripts/application.js

```diff
@@ -26,81 +26,97 @@
 //= require daterangepicker
 //= require_tree .
 
+function isMobileResolution() {
+  if ($(window).width() < 992) return true;
```

Comment:
I think you could avoid using those two returns in this function. What about doing something like this:
```javascript
function isMobileResolution() {
  return $(window).width() < 992;
}
```

Reply:
Yeah, it went unnoticed. I'll refactor this. Thank you!